guidelines/supportability.md

```diff
@@ -25,20 +25,26 @@ There are significant risks associated with exposing customers to Alpha-level up
 * The api/feature may change in a way that we cannot migrate customer workloads/configuration/resources
 * The api/feature may be removed entirely in a future k8s version leaving us having to carry patches or manually help customers to move forward which may even require an entirely new cluster installation
 
-Given these risks, alpha features **MUST NOT** be turned on downstream unless behind a feature gate that can only be enabled via the TechPreviewNoUpgrade 
-or CustomNoUpgrade featuresets.  If you/your team intend to enable (without a TechPreview gate) an upstream alpha feature, downstream, you must bring the request to 
-the staff engineering group for discussion.  This can be done by adding it to the agenda of an appropriate themed architecture calls (this may not get you approval 
-but it is a good place to start the conversation about what you want to do and why), or tagging `@aos-staff-engineers` on `#forum-arch`.  Approval won't be granted 
+Given these risks, alpha features **MUST NOT** be turned on downstream unless behind a feature gate that can only be enabled via the TechPreviewNoUpgrade
+or CustomNoUpgrade featuresets.  If you/your team intend to enable (without a TechPreview gate) an upstream alpha feature, downstream, you must bring the request to
+the staff engineering group for discussion.  This can be done by adding it to the agenda of an appropriate themed architecture calls (this may not get you approval
+but it is a good place to start the conversation about what you want to do and why), or tagging `@aos-staff-engineers` on `#forum-arch`.  Approval won't be granted
 without a set of people who clearly own working on the feature upstream and ensuring it will be driven to beta/GA state.
 
 Enabling an upstream alpha feature by putting it behind a TechPreviewNoUpgrade feature gate is acceptable as long as the team that is enabling it understands
 they are going to be responsible for helping debug issues associated with it.
 
 ## Enabling Upstream Beta features
 
-TBD - we'd like to disable these by default(and then have a relatively lower bar for enabling them), but right no there's no good way to disable the associated 
+TBD - we'd like to disable these by default (and then have a relatively lower bar for enabling them), but right no there's no good way to disable the associated
 upstream tests, so if we disabled them we'd break those tests w/o additional work to explicitly disable the associated tests.
 
+All of the upstream beta features (enabled by default) automatically fall under Tier 2 support as described in [our documentation](https://docs.openshift.com/container-platform/latest/rest_api/understanding-api-support-tiers.html#mapping-support-tiers-to-kubernetes-api-groups_understanding-api-tiers).
```

Comment:
Is this saying that all of the upstream beta features are enabled by default, or that of the upstream beta features the ones that are enabled by default are automatically Tier 2? I think the latter, in which case something like this parses a little more easily:

> Any upstream beta features that are are enabled by default in OpenShift fall under Tier 2 support, as described in ...

Reply:
first i think we need to distinguish between beta apis and beta features.

my understanding is:

beta features(that aren't tied to a beta api) are/can be enabled by default upstream (and i guess downstream as well since we can't disable them w/o breaking tests)

beta apis are not enabled by default upstream, nor are they enabled by default downstream.

So i'm not sure how it's obvious to a user that the non-api feature they are using is tier 2 (or even where we declare that beta features are effectively tier 2).  Does upstream have any useful messaging about this?